packages/app/src/templates/actions/update-template-order.action.ts

```diff
@@ -23,6 +26,10 @@ export const updateTemplateOrder = createAsyncThunk<Error | Template, Template>(
       )
     }
 
-    return data
+    if (error && error.data !== undefined) {
```

Comment:
This is not a change request, but rather a question/wondering.

We treat this kind of situation as a success operation - while in fact it is a partial success and a partial error. 

From high perspective - shouldn't we show the modal to the user that says more less something like "some of the templates were updated, but there was an error so please try again"?

Or what happens when there is no `successTemplates`, there are only `errorTemplates` - I think that the action should be rejected.

What do you think about that?

Reply:
Ok, I've rethought it and partial success should be treated as an error because it can break ordering. I changed this case to error in [43e06d1](https://github.com/mudita/mudita-center/pull/804/commits/43e06d134aad31f5e351a09965e5574fbf64604f)